config/lookoutingesterv2/config.yaml

```diff
@@ -0,0 +1,28 @@
+postgres:
+  maxOpenConns: 100
+  maxIdleConns: 25
+  connMaxLifetime: 30m
+  connection:
+    host: localhost
+    port: 5433
+    user: postgres
+    password: psw
+    dbname: postgres
+    sslmode: disable
+
+metrics:
+  port: 9003
+
+pulsar:
+  enabled: true
+  URL: "pulsar://localhost:6650"
+  jobsetEventsTopic: "persistent://armada/armada/events"
+
+paralellism: 1
```

Comment:
we don't need this- I had it in the old ingester, but the new framework only supports a single level of parallelism fornow.